src/lib/helpers.js

```diff
@@ -21,6 +21,17 @@ export const urlify = stringValue => {
   return 'undefined'
 }
 
+export const deurlify = stringValue => {
+  if (stringValue) return stringValue.split('_').join(' ')
+  return 'undefined'
+}
+
+export const makePascalCase = stringValue => {
+  if (!stringValue) return 'undefined'
+  if (stringValue.length > 1) return stringValue[0].toUpperCase() + stringValue.substr(1)
```

Comment:
```suggestion
 return stringValue[0].toUpperCase() + stringValue.substr(1)
```

in javascript, `'a'.substr(1) === ''`

in addition, if length is zero, toUpperCase returns the same string and `''.substr(1) === ''`

Reply:
changed it to

    return stringValue.charAt(0).toUpperCase() + stringValue.substr(1)

because `stringValue[0]` returns undefined if string is `''`, so toUpperCase would not work there